crime-applications-adaptor/application/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/PropertyMapper.java

```diff
@@ -175,32 +181,34 @@ String mapOtherRelation(Object otherRelationship) {
 
   ThirdPartyOwner.OwnerRelation mapOwnerRelation(PropertyOwner.Relationship relationship) {
     if (Objects.nonNull(relationship)) {
-      if (PropertyOwner.Relationship.OTHER.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.OTHER;
-      } else if (PropertyOwner.Relationship.EX_PARTNER.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.EX_PARTNER;
-      } else if (PropertyOwner.Relationship.BUSINESS_ASSOCIATES.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.BUSINESS;
-      } else if (PropertyOwner.Relationship.FAMILY_MEMBERS.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.FAMILY;
-      } else if (PropertyOwner.Relationship.FRIENDS.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.FRIENDS;
-      } else if (PropertyOwner.Relationship.HOUSE_BUILDER.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.BUILDER;
-      } else if (PropertyOwner.Relationship.HOUSING_ASSOCIATION.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.HOUSING_ASSOC;
-      } else if (PropertyOwner.Relationship.LOCAL_AUTHORITY.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.LOCAL_AUTH;
-      } else if (PropertyOwner.Relationship.PROPERTY_COMPANY.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.PPROPERTY_CO;
-      } else if (PropertyOwner.Relationship.PARTNER_WITH_A_CONTRARY_INTEREST.equals(relationship)) {
-        return ThirdPartyOwner.OwnerRelation.PARTNER_CONT;
+      switch (relationship) {
+        case OTHER:
+          return ThirdPartyOwner.OwnerRelation.OTHER;
+        case EX_PARTNER:
+          return ThirdPartyOwner.OwnerRelation.EX_PARTNER;
+        case BUSINESS_ASSOCIATES:
+          return ThirdPartyOwner.OwnerRelation.BUSINESS;
+        case FAMILY_MEMBERS:
+          return ThirdPartyOwner.OwnerRelation.FAMILY;
+        case FRIENDS:
+          return ThirdPartyOwner.OwnerRelation.FRIENDS;
+        case HOUSE_BUILDER:
+          return ThirdPartyOwner.OwnerRelation.BUILDER;
+        case HOUSING_ASSOCIATION:
+          return ThirdPartyOwner.OwnerRelation.HOUSING_ASSOC;
+        case LOCAL_AUTHORITY:
+          return ThirdPartyOwner.OwnerRelation.LOCAL_AUTH;
+        case PROPERTY_COMPANY:
+          return ThirdPartyOwner.OwnerRelation.PPROPERTY_CO;
```

Comment:
Is PPROPERTY_CO a typo? I see that is how it's spelt in the thirdPartyOwner.json, just want to make sure it's intentionally supposed to be spelt like that in the json so we don't get any problems when sending data to MAAT.

Reply:
Will make a not to follow up though!